### PR TITLE
Adding an example of release configuration

### PR DESCRIPTION
**Pull request by coladarci**
As per https://github.com/sorentwo/oban/issues/144 - while I didn't end up needing to change my `crontab` I did want to change concurrency of queues. 

If you think this is worth including, here it is!

Thanks again for your all your help recently.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -790,6 +791,79 @@ of Oban's telemetry events. As in the example above, attach it within your
 For more details on the default structured logger and information on event
 metadata see docs for the `Oban.Telemetry` module.
 
+## Release Configuration
+
+While having the same `Oban` configuration for every environment might be fine, there are certainly times you might
+want to make changes for a specific environment. Maybe you want to increase or decrease a queue's concurrency, for example.
+
+If you are using Elixir Releases, this is straight forward to do using [Moduel Config Providers](https://hexdocs.pm/mix/Mix.Tasks.Release.html#module-config-providers ).
```
Comment (maartenvanvliet) [inline README.md]:
```suggestion
If you are using Elixir Releases, this is straight forward to do using [Module Config Providers](https://hexdocs.pm/mix/Mix.Tasks.Release.html#module-config-providers ).
```

**Comment by milmazz** [comment]
> The reason I haven't merged it in is that I'm unsure about adding it to the README, it feels more like a guide.

I do agree with this, this is a great addition, but it describes a specific scenario for deployments. Given that Oban already depends on `ex_doc`, adding a "Guides" section it's easy. Let me know if you need help with this.

**Comment by sorentwo** [comment]
> I do agree with this, this is a great addition, but it describes a specific scenario for deployments. Given that Oban already depends on `ex_doc`, adding a "Guides" section it's easy. Let me know if you need help with this.

Yes, absolutely! I'm planning on a few initial guides (Installation, First Worker, Testing, Deployment) and would love help with any and all of them.

**Comment by milmazz** [comment]
@coladarci Now we have a `guides` section, maybe you should consider moving the _Release Configuration_ section into its own guide. More guides/recipes are in the works with #247 HTH

**Comment by sorentwo** [comment]
I've converted this into a guide and merged it into master. Thanks @coladarci 💛

**Pull request closed without merging**